### ISSUE TITLE
Unable to boot stuck on splash screen.

### ISSUE THREAD

**Issue by giodude143** [CrimsonMaple/Rei-Six#7]
Using FastBoot3DS with the latest release on O3DS 11.8.0-41U.
Please help, thanks.

**Comment by thetruecatboy** [comment]
Did you update to 1.2.1?
https://github.com/CrimsonMaple/Rei-Six/releases
1.2.1 has been tested and confirmed to be working on old3DS.

**Comment by giodude143** [comment]
Yes, and it just won't load. I tried to remove the /rei folder and deleted the boot.firm then downloaded the 1.2.1 but still won't load.

**Comment by thetruecatboy** [comment]
Are you running Rei-Six on your sysNAND or are you using an emuNAND?

**Comment by giodude143** [comment]
I'm not so sure but I just boot it using FastBoot3DS on the SD Card, sorry for being a noob.

I just followed the Recommended Setup and Instructions, don't know whats wrong.

**Comment by thetruecatboy** [comment]
Do you have a file called "loademunand" in the /rei folder?

**Comment by giodude143** [comment]
No, I created a loademunand file with no extension and tried again but this time it just shuts down automatically after showing the splash page after a few seconds.

found a debug.txt inside the /rei folder.
- External FIRM is invalid or corrupt. If this issue persists after reinstall, make an issue.

**Comment by thetruecatboy** [comment]
Well if that's files not there, then you aren't using an emuNAND.
Try referring to this bug report
https://github.com/CrimsonMaple/Rei-Six/issues/6

**Comment by giodude143** [comment]
I don't think that's the same, his using a new3ds while I have an OLD3DS and I also downloaded the boot.firm from this link https://www.dropbox.com/s/91jqzn0idsniasp/boot.firm?dl=0 but I still get the same message from the debug.txt after I tried to boot it again.

**Comment by thetruecatboy** [comment]
How did you update your 3DS?

**Comment by giodude143** [comment]
I updated it using the system update on the setting. 

Was running on luma 9.1 before.

**Comment by thetruecatboy** [comment]
Something has to be up with your system. I am currently running Rei-Six with the latest Fastboot beta and it's working just fine on my old3DS.
Make sure you delete the loademunand file if you aren't using an emuNAND. That's what throwing the external firm error
